README.md

```diff
@@ -29,6 +43,18 @@ export METADB_CONN_STR=postgresql://animl:animl@metadba.sea1.socrata.com:5432/bl
 
 ## Measuring relevance
 
+Once a job has completed, you can download the results and report NDCG by
+running the `download_crowdflower.sh` script. Before doing so, ensure that you
+have a a CrowdFlower API key and that a corresponding environment variable is
+set.
+
+```bash
+export CROWDFLOWER_API_KEY=LbcxvIlE3x1M8F6TT5hN
+python arcs/download_crowdflower.py -n -i 755163
```

Comment:
Note about how 755163 is just the current job number might be nice, just in case someone doesn't feel like running with -h?